src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -237,8 +238,16 @@ protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
             recipe.matches(true, importInventory, importFluids);
     }
 
+    protected int[] calculateOverclock(int EUt, int duration) {
```

Comment:
Returning an array to represent an arbitrary slice of data (where element 0 is one thing and element 1 is some other thing) feels really bad to me.  Is this idiomatic?